Source/Microsoft.Teams.Apps.FAQPlusPlus/Properties/Strings.resx

```diff
@@ -259,6 +263,14 @@ I am your friendly Q&amp;A bot that helps you provide support to people who inte
     <value>Share app feedback</value>
     <comment>Title text of share feedback user carousel card.</comment>
   </data>
+  <data name="ShareFeedbackSubmitText" xml:space="preserve">
```

Comment:
Where is this used?

Reply:
Please scrub the strings that you added and make sure they're actually used (e.g., AskAnExpertSubmitText above)

Reply:
We have removed unused strings. But the string which are used in manifest(name, short description etc) are still there

Reply:
Why do we need those in the resx?

Reply:
Deleted unused string from resx files. Thank you